docs/developer-guide/CLI Methods/generating-the-bundled-config-file.md

```diff
@@ -0,0 +1,87 @@
+# Generating the Bundled Configuration File
+
+[Running Basemaps Locally][running-basemaps-locally] \> [Configuring the Basemaps/server package][configuring-the-basemapsserver-package] \> [Providing a path to a bundled configuration JSON file][path-to-json-config-file] \> Generating the Configuration File
+
+This guide shows you how to generate the bundled configuration file using the **basemaps/cli** package.
+
+## Prerequisites
+
+### AWS Credentials
+
+You'll need **AWS credentials** with permission to read files from the **LINZ AWS S3** bucket at `s3://linz-basemaps`. These credentials will allow you to access and/or generate the bundled configuration file required for running the **Basemaps/server** package.
```

Comment:
```suggestion
You'll need **AWS credentials** with permission to read files from the **LINZ AWS S3** bucket at `s3://linz-basemaps`. These credentials will allow you to access and/or generate the bundled [basemaps-config](https://github.com/linz/basemaps-config) configuration file required for running the **basemaps/server** package.
```

to clarify the server can and will run without these credentials it is only if you want to run the server with linz/basemaps-config config you need access to the imagery.